### ISSUE TITLE
How can I add layers by myself

### ISSUE THREAD

**Issue by adwin5** [rcmalli/keras-vggface#1]
Thank you for the code.
I want to self define my fully connect layer, and use your include_top=False option.
But since your return is a object of Model(), I think I can not add layer. What should I do?

Basically, I want want to change the nb_class.
Thank you.

**Comment by rcmalli** [comment]
You can simply use this example code for custom classification model. I will also update the readme file.

~~~python
from keras.engine import Input, Model
from keras.layers import Flatten, Dense
from vggface import VGGFace

#custom parameters
nb_class = 2
hidden_dim = 512

image_input = Input(shape=(224, 224, 3))
vgg_model = VGGFace(input_tensor=image_input, include_top=False)
last_layer = vgg_model.get_layer('pool5').output
x = Flatten(name='flatten')(last_layer)
x = Dense(hidden_dim, activation='relu', name='fc6')(x)
x = Dense(hidden_dim, activation='relu', name='fc7')(x)
out = Dense(nb_class, activation='softmax', name='fc8')(x)
custom_vgg_model = Model(image_input, out)
~~~

**Comment by adwin5** [comment]
@rcmalli Thank you so much.
